DataSetExplorer/DataSetBuilder/Model/DataSetInstance.cs

```diff
@@ -1,17 +1,20 @@
 ﻿using CodeModel.CaDETModel.CodeItems;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
 using System.Linq;
 
 namespace DataSetExplorer.DataSetBuilder.Model
 {
     public class DataSetInstance
     {
-        public string CodeSnippetId { get; }
-        public string Link { get; }
-        public string ProjectLink { get; }
-        public SnippetType Type { get; }
-        public ISet<DataSetAnnotation> Annotations { get; }
+        public int Id { get; private set; }
+        public string CodeSnippetId { get; private set; }
+        public string Link { get; private set; }
+        public string ProjectLink { get; private set; }
+        public SnippetType Type { get; private set; }
+        public ISet<DataSetAnnotation> Annotations { get; private set; }
+        [NotMapped]
```

Comment:
Instead of polluting the domain model with persistence concerns, we should use the DBContext class to configure this.

![image](https://user-images.githubusercontent.com/7092212/126433058-9fd55f3b-efa6-40ed-9fef-9f33233d3f83.png)